### PR TITLE
Try to fix travis from tanking on db migrate?

### PR DESCRIPTION
**Pull request by murny**
Travis still failing on db migrate, says test database doesnt exist. DB create fails as well. Guessing DB container isnt fully initialized yet.

Maybe some extra sleep will help?

#153

### CODE REVIEW

**Comment by murny** [comment]
Hack for now...seems to fix it. If continues to happen, i say we rip this out and replace it with `docker-compose.lightweight`

As you said, better waiting an extra 15 secs then to manually restart the build all the time

**Comment by murny** [comment]
Proper solution is now done. Brought in a wait-for-it script as suggested by official docs:
https://docs.docker.com/compose/startup-order/

screen shot 2017-10-12 at 11 21 17 am

Seems to fix the issue.

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +10,9 @@ before_script:
   - docker-compose build web
   - docker-compose up -d web
 
-  - sleep 15 # wait for containers to finish setting up
   - docker ps
 
-  - docker-compose run --rm -e RAILS_ENV=test web rails db:create
+  - docker-compose run --rm -e RAILS_ENV=test web ./wait-for-it mysql:3306 --timeout=0 --strict -- rails db:create
```
Comment (cwant) [inline .travis.yml]: Maybe five minute timeout rather than no timeout?

**Pull request merged**